### DISCUSSION TITLE
Is there a way to prevent players from making sudo outposts by unclaiming chunks that are connecting other chunks to the homeblock.

### DISCUSSION THREAD

**Discussion by kevien12** [TownyAdvanced/Towny#7069] [Q&A] [answered]
I have players claiming out in straight lines from their towns, claiming a large area, then unclaiming the 1 chunk line connecting their actually town where the homeblock is from the new claimed area. This allows them to make sudo outposts getting past the outpost limit. Is there anyway to prevent players from unclaiming a chunk if that would mean other chunks would no longer be attached to the homeblock, or simply having all the chunks that are no longer connected to the homeblock unclaim?

**Comment by LlmDl** [comment, marked as answer]
In the config:
```
  # The minimum adjacent town blocks required to expand.
  # This can prevent long lines and snake-like patterns.
  # Set to -1 to disable. Set to 3 to force wider expansions of towns.
  min_adjacent_blocks: '-1'
``` 
Setting that to one will prevent them unclaiming.
